README.md

```diff
@@ -1 +1,219 @@
-# portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./logo.PNG" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
Kindly remove this part from your README file to keep it clean and professional.